using System;
using System.Collections.Generic;
using System.Data;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Moment6.Uppgift3
{
    internal class Program
    {
        static void Main(string[] args)
        {
            throw new Exception("felmeddelande");

            //här har jag kastat/thrown ett utantag/exception.
        }
    }
}
